index.css

```diff
@@ -65,14 +65,28 @@ h2 {
 }
 
 /**BUTTONS**/
-.reward-btn {
-    background-color: #9E4770;
+.reward-btns {
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    width: inherit;
+    
```

Comment:
Is an ending bracket missing here or is that just github weirdness?

Reply:
There is, it's all nested inside the .reward-btns for scoping and making sure none of my changes break out of that div. I do wish github showed that change...